### PR TITLE
Ddf-2902: Fixed multi-realm authentication

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Fixed multi-realm authentication for non idp realms.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@brendan-hofmann @ahoffer 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
@stustison [Security](https://github.com/orgs/codice/teams/security)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl 
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -29,10 +29,10 @@
             type="String"
             default="/=karaf"/>
 
-        <AD description="List of authentication types required for each context. List of default valid authentication types are: IDP, SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
+        <AD description="List of authentication types required for each context. List of default valid authentication types are: SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
             name="Authentication Types" id="authenticationTypes" required="true" cardinality="100"
             type="String"
-            default="/=IDP|GUEST,/solr=SAML|PKI|BASIC"/>
+            default="/=SAML|GUEST,/admin=SAML|basic,/system=basic,/solr=SAML|PKI|basic,/sources=SAML|basic,/security-config=SAML|basic"/>
```
Comment (coyotesqrl) [inline platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml]: ❓ Change to the metatype not reflected in the blueprint as well (https://github.com/AzGoalie/ddf/blob/bad47c3254d6f08a43cab2a2dcfdc7992ccdf81d/platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L26). Should they be synched?

Comment (stustison) [inline platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml]: This change shouldn't be here at all @AzGoalie

Comment (AzGoalie) [inline platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml]: Fixed

**Comment by AzGoalie** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by AzGoalie** [comment]
build now

**Pull request merged**